Update 11_taxonomic_information.qmd

jkc9886 opened the pull request:
Grammatical and Punctuation corrections.

jkc9886 committed:
Update 11_taxonomic_information.qmd

Grammatical and Punctuation corrections.

antagomir commented on inst/pages/11_taxonomic_information.qmd:
```diff
@@ -120,10 +120,10 @@ head(getUnique(tse, rank = "Phylum"))
 ### Generate a hierarchy tree on the fly {#sec-fly-tree}
 
-A hierarchy tree is not an actual phylogenetic tree.
+A hierarchy tree is not an actual phylogene tic tree.
```
space typo?

antagomir commented on inst/pages/11_taxonomic_information.qmd:
```diff
@@ -336,22 +336,22 @@ references.
 
- * 'z' Z transformation scales data to zero mean and unit variance;
-   this is used to bring features (or samples) to more comparable
+ * 'z' Z transformation scales data to zero mean and unit variance.
```
This should be updated to "scale" (we renamed it recently).

antagomir replied:
Sorry, in fact it is "standardize" (and not "scale") I think. In the latest mia version.

jkc9886 committed:
Update 11_taxonomic_information.qmd

"z" transformation --> "Standardize" Tranformation. Is this what you meant @antagomir ?

jkc9886 committed:
Update 11_taxonomic_information.qmd

lower case/upper case changes

TuomasBorman commented on inst/pages/11_taxonomic_information.qmd:
```diff
@@ -329,29 +329,29 @@ references.
- * 'z' Z transformation scales data to zero mean and unit variance;
-   this is used to bring features (or samples) to more comparable
-   levels in terms of mean and scale of the values. This can enhance
+ * 'standardize' Standardize transformation scales data to zero mean and 
+   unit variance. This is used to bring features (or samples) to more 
+   comparable levels in terms of mean and scale of the values. This can enhance
```
Add "(or Z-transformation)"

antagomir replied:
I would prefer not referring to z-transformation any more since it has different common meaning.

TuomasBorman replied:
Ok, that is fine. I guess Z-score is relatively common though. The description already states clearly what the transformation does, but for someone Z-score might be more familiar term as "standardization" can also mean different things

antagomir replied:
Can you point us to at least 1-3 references where other than us use z transformation in our meaning in microbiome analyses?

TuomasBorman replied:
With a quick search (I have not done that earlier; my point was that it might be good to list common synonyms from basic statistics even though they are not commonly used in microbiome research.)

- https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-016-0222-x/figures/2
- https://www.researchgate.net/publication/338713495_Diversity_of_Bacterial_Community_in_the_Oxygen_Minimum_Zones_of_Arabian_Sea_and_Bay_of_Bengal_as_Deduced_by_Illumina_Sequencing/figures?lo=1
. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8250139/

antagomir replied:
This also discusses "z-score" in the same meaning: https://en.wikipedia.org/wiki/Standard_score

The "z" has multiple meanings like "z score", "z value" "z factor" etc.
https://en.wikipedia.org/wiki/Z-score_(disambiguation)

Seems fine to refer to this also as z-score.

antagomir committed:
Merge branch 'devel' into jkc9886-patch-9

jkc9886 committed:
Update 11_taxonomic_information.qmd

Edited line 339: Standardize( or 'z-score') transformation

jkc9886 committed:
Merge branch 'devel' into jkc9886-patch-9

jkc9886 commented:
This PR is ready to be merged.

antagomir committed:
Merge branch 'devel' into jkc9886-patch-9

antagomir reviewed (approved):
Good to me!

antagomir commented:
This seems OK - to merge, @TuomasBorman ?

TuomasBorman committed:
Merge branch 'devel' into jkc9886-patch-9
